content/USER_GUIDE.md

```diff
@@ -0,0 +1,242 @@
+
+---
+
+## **4️⃣ Viewing Results**
+
+1. Once the workflow is finished, check the **log messages**.
+2. Navigate to the **"Viewer"** tab in the sidebar to analyze the deconvoluted data.
+3. If needed, **download results** by clicking **"Download Files"**.
+   (Will be explained later in step 7 and 8 in this guide)
+
+---
+
+## **5️⃣ Manual Result Upload**
+
+1. Click on the **"Manual Result Upload"** tab.
+2. Upload FLASHDeconv output files (`*_annotated.mzML` & `*_deconv.mzML`) or TSV files for ECDF Plot.
+3. Browse files or **drag and drop** them into the upload section.
+4. Click **"Add files to workspace"** to finalize.
+
+
+![Manual Upload](/static/Images/flashdeconv_manual_upload.png)
+
+---
+
+## **6️⃣ Using Example Data**
+
+1. Click the **"Example Data"** tab.
+2. Click **"Load Example Data"** to use the preloaded dataset.
+3. The example data will appear in the uploaded experiments table.
+
+ 
+![Example Data](/static/Images/flashdeconv_example_data.png)
+
+---
+
+## **7️⃣ Layout Manager**
+
+The **Layout Manager** allows users to customize the experiment display settings.
+
+1. Select the **number of experiments** to view at once.
+2. Click **"Select..."** to choose components to add:
+   - **MS1 raw heatmap**
+   - **MS1 deconvolved heatmap**
+   - **Scan table**
+   - **Deconvolved spectrum**
+   - **Annotated spectrum**
+   - **Mass table**
+   - **2D plot**
+3. Click **Save** to apply changes.
+
+
+![Layout Manager](/static/Images/flashdeconv_layout_manager.png)
+
+---
+
+## **8️⃣ Viewing Results in FLASHViewer**
+
+1. Navigate to the **Viewer** tab.
+2. Choose an experiment from the dropdown.
+3. View the selected one from Layout manager: scan table, mass table, annotated spectrum, and deconvolved spectrum etc.
+
+![FLASHViewer](/static/Images/flashdeconv_viewer.png)
+
+---
+
+## **9️⃣ Downloading Results**
+
+1. Navigate to the **Download** tab.
+2. Locate the experiment you want to download.
+3. Click **"Prepare Download"** to generate the downloadable files.
+4. To delete an experiment, click the **trash icon** next to the experiment name.
+
+![Download Results](/static/Images/flashdeconv_download.png)
+
+---
+
+
+
+# **FLASHTnT Guide**
+
+## **1️⃣ Uploading MS Data & Database**
+
+1. Navigate to **FLASHTnT > Workflow** in the sidebar.
+2. Click **"File Upload"** to upload your `.mzML` file.
+
+![Download Results](/static/Images/flashTnT_upload.png)
+
+3. Click the **"Database"** tab to upload the necessary **FASTA** database files.
+4. Click **"Add Database"** to confirm the upload.
+
+
+![Download Results](/static/Images/flashTnT_databaseupload.png)
+---
+## **2️⃣ Configuring Parameters**
+1. Click the **"Configure"** tab.
+2. Select your uploaded **mzML file**.
+3. Choose the **FASTA database** file.
+4. There are two sub-tabs for configuring parameters: **FLASHDeconv** and **FLASHTnT**.
+5. Adjust FLASHTnT parameters:
+Modify Ex Parameters (max_mod_mass, max_mod_count).Adjust **general settings** such as:
+   - Threads
```

Comment:
Maybe one sentence for the exemplary parameters. For threads there could be a note that this can only be modified in local settings.